src/test/java/de/numcodex/feasibility_gui_backend/query/broker/direct/DirectBrokerClientFlareIT.java

```diff
@@ -20,18 +20,17 @@
 import static de.numcodex.feasibility_gui_backend.query.collect.QueryStatus.FAILED;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.*;
-import static org.springframework.http.HttpHeaders.ACCEPT_ENCODING;
 import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
 import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
 
 @ExtendWith(MockitoExtension.class)
 @SuppressWarnings("NewClassNamingConvention")
-class DirectBrokerClientIT {
+class DirectBrokerClientFlareIT {
```

Comment:
Why is the non-standard header `internal/json` used?

Reply:
Don't know...just renamed the file

Reply:
I guess it's just a remnant from earlier versions. Replacing it the default application/json mediatype works.

Reply:
Ok Thanks.